public/agp/resources/abstract-bergenstal.html

```diff
@@ -212,7 +212,7 @@
         .marque_effect {
             background-color: #000;
             color: #fff;
-            font-size: 18px;
+            font-size: 110%;
```

Comment:
@digimagic - On some of these you're switching from 110% to 18px but in others you're switching from 18px to 110%. Is this intentional?